lib/libutee/include/tee_api_defines.h

```diff
@@ -79,6 +79,7 @@
 #define TEE_PARAM_TYPE_VALUE_INPUT      1
 #define TEE_PARAM_TYPE_VALUE_OUTPUT     2
 #define TEE_PARAM_TYPE_VALUE_INOUT      3
+#define TEE_PARAM_TYPE_MEMREF_SECURE    4
```

Comment:
I used 4 but we could use IDs above 0x9 (the last `MEMREF_INOUT.`) up to 0xFF (accoring to the GP specs and current optee implementation).

If we really define a new ID, i would feel strange having some `TEE_PARAM_TYPE_xxx` in the `tee_api_defines.h` and others in `tee_api_defines_extentions.h.` Yet i'm ok to do so.